src/resolve/resolved-dns-cache.c

```diff
@@ -1265,6 +1268,11 @@ int dns_cache_export_shared_to_packet(DnsCache *cache, DnsPacket *p) {
                         if (!j->shared_owner)
                                 continue;
 
+                        /* RFC6762 7.1: Don't append records with less than
+                         * half the TTL remaining as known answers. */
```

Comment:
Not a big matter, but it is not necessary to wrap comment so frequently. We allow about 109ch per line.